samples/boards/stm32/pm_lptim/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _Power-Management-LP-Timer:
+
+Power Management LP Timer
+#########################
+
+Overview
+********
+
+This is a simple application which enter/exit the low power mode.
+The led is ON in running mode and is off during low power periods.
+
+.. _Power-Management-LP-Timer-requirements:
+
+Requirements
+************
+
+You will see this error if you try to build sample for an unsupported board:
+
+.. code-block:: none
+
+   Unsupported board: led0 devicetree alias is not defined
+   Unsupported board: sw0 devicetree alias is not defined
+
+The board must have a user button and a LED connected via GPIO pins. These are called "User
+LEDs" on many of Zephyr's :ref:`boards`. The Button and LED must be configured using the
+``sw0`` :ref:`devicetree <dt-guide>` alias and
+``led0`` :ref:`devicetree <dt-guide>` alias.
+This is usually done in the :ref:`BOARD.dts file <devicetree-in-out-files>` or a :ref:`devicetree overlay
+<set-devicetree-overlays>`.
+
+Building and Running
+********************
+
+Build and flash sample as follows, changing ``nucleo_l476rg`` for your board:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/stm32/pm_lptim
+   :board: nucleo_l476rg
+   :goals: build flash
+   :compact:
+
+After flashing, the LED is on, waiting for entering the low Power mode.
+After WAIT_TIME short delay, the kernel is entering the low power mode (stm32 stop mode),
```

Comment:
Please don't make the user search the code for what `WAIT_TIME` and `SLEEP_TIME` mean.  Write "a half second delay" and "for four seconds".  Nobody's likely to modify the code to other durations.

Reply:
Not really sure it is worth to duplicate timing values from code to text, this is just a readme  'FYI'.

Reply:
It really is worth it.  The new user assessing whether something is working will be looking at the README to see what the application is supposed to do.  Please, do not make them look in multiple places to get an accurate understanding of expected behavior.  It will needlessly annoy a potential Zephyr user.

Reply:
done